code/modules/recycling/disposal/loafer.dm

```diff
@@ -0,0 +1,155 @@
+/obj/item/reagent_containers/food/snacks/prison_loaf
+	name = "prison loaf"
+	desc = "A rather slapdash loaf designed to feed prisoners.  Technically nutritionally complete and edible in the same sense that potted meat product is edible."
+	icon = 'icons/obj/food/loafing_about.dmi'
+	icon_state = "ploaf0"
+	force = 0
+	throwforce = 0
+	volume = 1000
+	list_reagents = list("nutriment" = 3, "flour" = 2, "corn_starch" = 10, "mayonnaise" = 10, "mushroomhallucinogen" = 3, "synthflesh" = 10)
+	var/loaf_factor = 1
+	var/loaf_recursion = 1
+
+/obj/item/reagent_containers/food/snacks/prison_loaf/proc/loafing_it_up()
+	var/orderOfLoafitude = max( 0, min( round( log(8, loaf_factor)), 10 ) )
+	w_class = min(orderOfLoafitude+1, 4)
+	switch(orderOfLoafitude)
```

Comment:
I'll need you to graph the required amount for each density

Reply:
That's spoiling the fun of loafistry

Reply:
prelim math, 8000~ loaf factor for like, 4.8 loafitude